content/about/agreements.md

```diff
@@ -138,6 +138,20 @@ We agree to have quarterly check-ins to facilitate more long-term/career guidanc
 
 To build community and deeper inter-personal connections, and to have fun, we agree to having semi-annual retreats,  typically during the week before classes start each semester (this likely means 1 beach retreat, and 1 ski retreat).  The retreats will include a celebration of past successes, as well as a discussion of collective goals for the subsequent year. These retreats will be organized by trainees.
 
+#### Summer Community Outreach
+
+Every summer, we will engage in summer mentorship in partnership with some Baltimore public education institution (e.g. state university, public high school). Examples include:
```

Comment:
who is the "we" in this statement - as in how many of us are committing to doing this?

Reply:
good point, maybe "funded lab members including PhD students and staff?"